src/content/1.7/modules/core-updates/1.7.7.md

```diff
@@ -40,3 +147,12 @@ New hook                       | Function
 **displayAdminOrderSide**      | It is displayed in the side column (similar to displayAdminOrderSideBottom) but right under the customer view
 **displayAdminOrderMain**      | It is displayed in the side column (similar to displayAdminOrderMainBottom) but right under the details view
 **actionGetAdminOrderButtons** | It is used to build a collection of buttons in the order thanks to an `ActionsBarButtonsCollection` which is automatically rendered
+
+## Other changes
+
+- CLDR `Specification\Number` [is now immutable](https://github.com/PrestaShop/PrestaShop/pull/15755)
+- Configuration forms have been split up [for security reasons](https://github.com/PrestaShop/PrestaShop/pull/17482)
+- New alias for webpack namespace app -> @app
```

Comment:
Also we introduce modules namespace, dont know if worth mentioning

Reply:
I don't think it's worth it, that's why I left it out